content/resources/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Resources"
+date: 2020-07-08T10:00:45-04:00
+layout: single
```

Comment:
I think we need to include the videos here in the same way we are doing it on iot.eclipse.org/resources

Right now the page is blank.